sample_command_policy.json

```diff
@@ -0,0 +1,28 @@
+{
+  "commands" : [
+    [ 
+        "cd examples/c_code_proj;  ./makefile",
+        "ls -ltr examples/c_code_proj/test",
+        "build_snap",
+        "default_policy.json"
+    ],
+    [
+        "cd examples/java_maven_proj;  mvn clean",
+        "ls -ltr examples/java_maven_proj/target",
+        "build_main",
+        "README.md"
+    ],
+    [
+        "cd tuf; sudo python setup.py install",
+        null,
+        "build_tuf_install",
+        null
+    ],
+    [
+        "cd examples/c_code_proj;  ./makefile_err",
+        "ls -ltr testErr",
+        "build_snapErr",
+        null
+    ]
+  ]
+}
```

Comment:
Ok, I understand what you're saying about clarity and simplicity. To me though, it seems clearer and simpler to use strings as keys here instead of position. That way, a glance at one of these files can give you a lot more info. @SantiagoTorres do you have any thoughts on this? Is there a best-practices standard here, or is it just more stylistic?